### PR TITLE
Split Boots of Speed off of Boots of Levitation.

### PR DESCRIPTION
**Pull request by Pvanduyse**

### CODE REVIEW

File: source/equipment/magic-items/boots-of-speed.rst [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.. _srd:boots-of-levitation:
```
Comment (maxrothman) [inline source/equipment/magic-items/boots-of-speed.rst]: This should match the name of the item

**Comment by maxrothman** [comment]
Thanks again for contributing! A few notes:
* Did you also remove the "boots of speed" block from the "boots of levitation" file? I don't see that diff.
* The test failure is being caused by the label at the top of the file being wrong. Once you correct it, the tests should pass.

**Comment by Pvanduyse** [comment]
I'm making a fairly large commit at the moment that fixes all of these issues in the magical equipment section. I'll have it done soon, then commit it all at once.

**Comment by maxrothman** [comment]
Wow, thanks!

**Pull request closed without merging**